kiez/kiez.py

```diff
@@ -83,9 +87,9 @@ def __init__(
         elif n_neighbors <= 0:
             raise ValueError(f"Expected n_neighbors > 0. Got {n_neighbors}")
         self.n_neighbors = n_neighbors
-        self.algorithm = (
-            SklearnNN(n_candidates=n_neighbors) if algorithm is None else algorithm
-        )
+        if algorithm is None and algorithm_kwargs is None:
+            algorithm_kwargs = dict(n_candidates=n_neighbors)
```

Comment:
should the `n_candidates` always be set to `n_neighbors` if it's instantiating from a class? I think `n_neighbors` is shared by all algorithm classes, right?

Reply:
if that's the case, I will change it to be 
```
if algorithm_kwargs is None:
    algorithm_kwargs = {}
algorithm_kwargs.setdefault('n_candidates', n_neighbors)
```

Reply:
so if you make a custom algorithm with n_candidates=100, what should get passed to the Kiez class? always 100? or it can still be anything?

If it always has to be the same, maybe it's better to look up this number from the algorithm instance itself rather than having it as an argument to the Kiez class